src/main/java/com/adobe/commerce/cif/model/cart/ShippingInfo.java

```diff
@@ -27,10 +27,10 @@
     protected String name;
     
     @ApiModelProperty(value = "The shipping price.", required = true)
-    protected Price price;
```

Comment:
I recommend renaming the field name to `cost`, since shipping fee is a cost you pay, not a price.